test/extensions/filters/common/rbac/matchers_test.cc

```diff
@@ -101,6 +102,15 @@ TEST(OrMatcher, Permission_Set) {
 
   checkMatcher(RBAC::OrMatcher(set), false, conn, headers, info);
 
+  perm = set.add_rules();
+  envoy::type::v3::Int32Range range;
+  range.set_start(123);
+  range.set_end(456);
+  perm->mutable_destination_port_range()->set_start(123);
+  perm->mutable_destination_port_range()->set_end(456);
```

Comment:
nit: why not copy `range` into `destination_port_range`?

Reply:
That's right. Unused variable has been removed.